string[] keys
float32[] values
---
uint8 status
